### PR TITLE
Add Support for Travel Restrictions API

### LINKED ISSUE #119: Add support for Travel Restrictions API

### PR DESCRIPTION
**Pull request by siddydutta**
Fixes #119

## Changes for this pull request

- Added support for [Travel Restrictions API ](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions/api-reference)
- Added tests and docs

### CODE REVIEW

**Comment by tsolakoua** [comment]
@siddydutta thanks a lot for your contribution! Could you please update your PR with the last changes of the master branch?

File: README.rst [hunk 1]
```diff
@@ -336,6 +336,9 @@ List of supported endpoints
     amadeus.analytics.itinerary_price_metrics.get(originIataCode='MAD', destinationIataCode='CDG',
                                                 departureDate='2021-03-21')
 
+    # Travel Restrictions
+    amadeus.travel_restrictions.covid19_area_report.get(countryCode="US")
```
Comment (tsolakoua) [inline README.rst:340]: The SDK maps every API path to a similar path. For example, `/v1/duty-of-care/diseases/covid19-area-report`
should be represented as`amadeus.duty_of_care.diseases.covid19_area_report`.
So the classes `DutyOfCare` and `Diseases` have to be created and delete the `TravelRestrictions`.

**Review by tsolakoua — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (siddydutta) [reply README.rst:340]: Ah, understood, will make those changes.

**Review by tsolakoua — APPROVED** [review]
Thank you @siddydutta!!

**Pull request merged**